docs/core/build/distribution-packaging.md

```diff
@@ -17,49 +17,109 @@ ms.workload:
 
-#### Preview versions
+- (4) **sdk/NuGetFallbackFolder** contains a cache of NuGet packages used by an SDK during the `dotnet restore` step.
 
-Package maintainers may decide to include preview versions of the shared framework and SDK. Those should never be included in the unversioned `dotnet-sdk` package, but may be released as versioned packages with an additional preview marker appended to the major and minor version sections of the name. For example, there may be a `dotnet-sdk-2.0-preview-final` package.
+The **shared** folder contains frameworks. A shared framework provides a set of libraries at a central location so they can be used by different applications.
 
-### Optional additional packages
+- (5) **shared/Microsoft.NETCore.App/\<runtime version>** This framework contains the .NET Core runtime and supporting managed libraries.
 
-Some maintainers may choose to provide additional packages such as:
+- (6,7) **shared/Microsoft.AspNetCore.{App,All}/\<aspnetcore version>** contains the ASP.NET Core libraries. The libraries under `Microsoft.AspNetCore.App` are developed and supported as part of the .NET Core project. The libraries under `Microsoft.AspNetCore.All` are a superset which also contains 3rd party libraries.
 
-* `dotnet-lts`: the latest Long-Term Support (LTS) version of the shared framework. [LTS and Current release trains](~/docs/core/versions/lts-current.md) correspond to different cadences at which .NET Core gets released. Users have a choice of adopting one or the other train, based on how often they are willing to update. This is a concept that is also tied to support levels, so it may or may not make sense depending on the distro being considered.
+- (8) **LICENSE.txt,ThirdPartyNotices.txt** are the .NET Core license and licenses of third-party libraries used in .NET Core.
 
-## Disk layout
+- (9,10) **dotnet.1.gz, dotnet** `dotnet.1.gz` is the dotnet man page. `dotnet` is a symlink to the dotnet host(1). These files are installed at well known locations for system integration.
+
+## Recommended packages
+
+.NET Core versioning is based on the runtime component `[major].[minor]` version numbers.
+The SDK version uses the same `[major].[minor]` and has an independent `[patch]` which combines feature and patch semantics for the SDK.
+For example: SDK version 2.2.302 is a the 2nd patch release of the 3rd feature release of the SDK that supports the 2.2 runtime.
+
+Some of the packages include part of the version number in their name. This allows the end-user to install a specific version.
+The remainder of the version is not included in the version name. This allows the OS package manager to update the packages (e.g. automatically installing security fixes).
```

Comment:
OS package manager is Linux specific today. We don't have anything similar on Mac (unless a user is using something like Brew) or on Windows (unless a user is using something like chocolatey). However, in both cases, these are "unofficial" acquisition mechanism.